### PR TITLE
add italian and bulgarian translation

### PR DESCRIPTION
**Pull request by BlueButterflies**
Hi, I added translation in Italian and Bulgarian languages.

Thank you in advance

### CODE REVIEW

**Comment by BlueButterflies** [comment]
Hi, I added translation in Italian and Bulgarian languages.

Thank you in advance

**Comment by BlueButterflies** [comment]
I don't understand  where is the problem .

**Comment by ahpook** [comment]
```
/home/runner/work/opensourcefriday/opensourcefriday/vendor/gems/ruby/3.1.0/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:143:in `parse': (/home/runner/work/opensourcefriday/opensourcefriday/_data/locales/it.yml): did not find expected key while parsing a block mapping at line 103 column 7 (Psych::SyntaxError)
```
I ran the file through yamlpp-highlight and found the error. There is an `'` in line 105 that makes the parse fail - can you change this to a `ʼ` (UTF-8 apostrophe)? 

```
    Will_Norris:
      title: "Open Source su <a href='https://github.com/google'>Google</a>"
      name: "Will Norris"
      message: 'Contribuire a progetti open source continua a essere uno dei progetti più apprezzati dal 20% degli ingegneri di Google. L'open source svolge un ruolo così importante in gran parte di ciò che facciamo, quindi cerchiamo di rendere <a href="https://opensource.google.com/docs/patching/"> il processo</a> il più semplice possibile possibile.'

**Comment by ahpook** [comment]
A couple of the other `message` keys also have `'` characters that will cause the same problem.

It is because the whole YAML value is wrapped in a single quote `message: 'the message here'` - but when the content also has a `'`, YAML thinks the value is closed:  `message: 'L'open source` will only see `L` , and then start parsing the next key at `open`. Using a `’` will get around the problem, and will let the embedded URLs parse correctly too.

File: _data/locales/it.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    Chris_Lamb:
+      title: "Responsabile del <a href='https://www.debian.org'>progetto Debian</a>"
+      name: "Chris Lamb"
+      message: "Sono arrivato dove sono oggi contribuendo al software open source, permettendomi non solo di migliorare le mie capacità di sviluppatore di software ma anche di aumentare la mia rete di colleghi e di incontrare nuovi e interessanti amici provenienti da culture estremamente diverse. Lo consiglio vivamente."
+    Will_Norris:
+      title: "Open Source su <a href='https://github.com/google'>Google</a>"
+      name: "Will Norris"
+      message: 'Contribuire a progetti open source continua a essere uno dei progetti più apprezzati dal 20% degli ingegneri di Google. L'open source svolge un ruolo così importante in gran parte di ciò che facciamo, quindi cerchiamo di rendere <a href="https://opensource.google.com/docs/patching/"> il processo</a> il più semplice possibile possibile.'
```
Comment (ahpook) [inline _data/locales/it.yml:105]:
```suggestion
      message: 'Contribuire a progetti open source continua a essere uno dei progetti più apprezzati dal 20% degli ingegneri di Google. L’open source svolge un ruolo così importante in gran parte di ciò che facciamo, quindi cerchiamo di rendere <a href="https://opensource.google.com/docs/patching/"> il processo</a> il più semplice possibile possibile.'
```

File: _data/locales/it.yml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    Christine_Abernathy:
+      title: "Open Source su <a href='https://github.com/facebook/'>Facebook</a>"
+      name: "Christine Abernathy"
+      message: "L'open source è in linea con la nostra missione di rendere il mondo più aperto e connesso. Lavorare con la comunità ci dà l'opportunità di condividere il nostro lavoro e le sfide che miriamo a risolvere e incoraggia le persone di talento a contribuire con le loro idee e i loro sforzi per far progredire il mondo."
```
Comment (ahpook) [inline _data/locales/it.yml:109]:
```suggestion
      message: 'L’open source è in linea con la nostra missione di rendere il mondo più aperto e connesso. Lavorare con la comunità ci dà l’opportunità di condividere il nostro lavoro e le sfide che miriamo a risolvere e incoraggia le persone di talento a contribuire con le loro idee e i loro sforzi per far progredire il mondo.'
```

File: _data/locales/it.yml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    Rich_Harris:
+      title: "<a href='https://github.com/rollup/rollup'>Rollup</a> e <a href='https://github.com/sveltejs/svelte'>Svelte</a> manutentore"
+      name: "Rich Harris"
+      message: "Ciò che manca alla maggior parte dei progetti open source non è il denaro, ma il tempo. Molti manutentori hanno un lavoro a tempo pieno e non possono realisticamente ridurre le proprie ore. Ciò di cui quasi tutti i progetti hanno bisogno sono più contributori. L'Open Source Friday è un modo perfetto per le aziende di supportare l'ecosistema open source da cui dipendono in modo significativo."
```
Comment (ahpook) [inline _data/locales/it.yml:113]:
```suggestion
      message: "Ciò che manca alla maggior parte dei progetti open source non è il denaro, ma il tempo. Molti manutentori hanno un lavoro a tempo pieno e non possono realisticamente ridurre le proprie ore. Ciò di cui quasi tutti i progetti hanno bisogno sono più contributori. L’Open Source Friday è un modo perfetto per le aziende di supportare l’ecosistema open source da cui dipendono in modo significativo."
```

File: _data/locales/it.yml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    Brett_Cannon:
+      title: "<a href='https://github.com/python/cpython'>Python core</a> sviluppatore"
+      name: "Brett Cannon"
+      message: 'Una volta che sei manutentore di un progetto su cui fanno affidamento altri, l'open source passa dalla "condivisione del codice" alla "condivisione della manutenzione". Come tutti sappiamo, la "manutenzione" richiede tempo che viene speso in modo più efficace in un blocco continuo per assicurarsi di poterlo completare, ad es. revisione del codice prima di iniziare a perdere il contesto.'
```
Comment (ahpook) [inline _data/locales/it.yml:117]:
```suggestion
      message: 'Una volta che sei manutentore di un progetto su cui fanno affidamento altri, l’open source passa dalla "condivisione del codice" alla "condivisione della manutenzione". Come tutti sappiamo, la "manutenzione" richiede tempo che viene speso in modo più efficace in un blocco continuo per assicurarsi di poterlo completare, ad es. revisione del codice prima di iniziare a perdere il contesto.'
```

**Review by ahpook — CHANGES_REQUESTED** [review]
I think this will permit the tests to pass.

File: _data/locales/it.yml (hunk 3 above)
Comment (BlueButterflies) [reply _data/locales/it.yml]: I apologize for the delay. I made the requested changes.

**Comment by BlueButterflies** [comment]
I apologize for the delay. I made the requested changes.

**Comment by ahpook** [comment]
Excellent, looks like all the tests are passing now. Thank you for all this work @BlueButterflies !

**Pull request merged**